api/admin/model/dashboard_statistics.py

```diff
@@ -0,0 +1,143 @@
+from typing import Dict, List, Type, TypeVar
+
+from pydantic import BaseModel, Extra, Field, NonNegativeInt
+
+
+def _snake_to_camel_case(name: str) -> str:
```

Comment:
I thought about putting it somewhere else, but no place really made a lot of sense to me at the moment, since it's used in only one place at the moment. If we're going to use it in more places, I think it would make sense to pull it out.